ui/app/routes/vault/cluster.js

```diff
@@ -29,7 +30,10 @@ export default Ember.Route.extend(ModelBoundaryRoute, ClusterRoute, {
 
   model(params) {
     const id = this.getClusterId(params);
-    return this.get('store').findRecord('cluster', id);
+
+    return this.get('version').fetchFeatures().then(() => {
```

Comment:
Could move this to a `beforeModel` hook, to prevent the explicit promise chaining and keep the `model` hook simple.

Reply:
Oh yep, good call - updated in e9721df. Thanks!